### PR TITLE
Virustotal dbotscore duplication fix

### PR DESCRIPTION
**Pull request by roysagi**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18022

## Description
Added VirusTotal 5.5 - now able to use DT with DbotScore.

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](link)
- [ ] [CHANGELOG](link)

### CODE REVIEW

**Comment by roysagi** [comment]
To easily understand the fix I only changed:
ec.DBotScore => ec["DBotScore(val.Indicator && val.Indicator === obj.Indicator && val.Vendor === obj.Vendor && val.Type === obj.Type)"] wherever it appeared.

@guykeller - to streamline your CR
@mayagoldb -FYI

**Comment by roysagi** [comment]
Also a question - though it looks like I added a new integration this has the same features and descriptions as VirusTotal.
Also the bug fix will only take affect from Demisto 5.5 - so I don't know if it should appear in current release notes.

File: Integrations/integration-VirusTotal_5.5.yml [hunk 1]
```diff
@@ -0,0 +1,1317 @@
+commonfields:
+  id: VirusTotal 5.5
```
Comment (anara123) [inline Integrations/integration-VirusTotal_5.5.yml]:
```suggestion
  id: VirusTotal
```

File: Integrations/integration-VirusTotal_5.5.yml [hunk 2]
```diff
@@ -0,0 +1,1317 @@
...
+  version: -1
+name: VirusTotal 5.5
```
Comment (anara123) [inline Integrations/integration-VirusTotal_5.5.yml]:
```suggestion
name: VirusTotal
```

File: Integrations/integration-VirusTotal.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ commonfields:
   version: -1
 name: VirusTotal
 display: VirusTotal
+toversion: 5.5.0
```
Comment (anara123) [inline Integrations/integration-VirusTotal.yml]:
```suggestion
toversion: 5.1.0
```

File: Integrations/integration-VirusTotal_5.5.yml [hunk 4]
```diff
@@ -0,0 +1,1317 @@
...
+display: VirusTotal 5.5
```
Comment (anara123) [inline Integrations/integration-VirusTotal_5.5.yml]:
```suggestion
display: VirusTotal
```

**Comment by anara123** [comment]
@roysagi please create self service with demisto master and upload the content.zip to make sure that VT is uploaded and everything works properly

**Comment by ronykoz** [comment]
@roysagi before merging please do a diff between the VT we have in master to check that only the DTs were updated

**Comment by dantavori** [comment]
The previous version of the 5.5 integration didn't contain the last update in VT (batch support for generic commands) and the TPB tested these changes. That's the reason for the build failure. Now it is fixed.

**Comment by dantavori** [comment]
Ran `!ip` multiple times in a self-service instance, and it seems to do the job (incidents are created and there are no duplicates).
FYI, I got this error once: `No content received. Possible API rate limit reached.`. From searching in Slack it seems that our VT instance isn't very stable, just want to make sure.

**Comment by ronykoz** [comment]
@anara123 so are we ok with this?

**Comment by anara123** [comment]
Lets merge then. Make sure that in release notes we mention that this fix
will be available only from 5.5

**Comment by ronykoz** [comment]
@roysagi lets add changelog and state that the fix is only from Demisto version 5.5 please

**Comment by roysagi** [comment]
Added RN and also made sure again that all the changes against the version in master are of DT only - all good.

File: Integrations/integration-VirusTotal_CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
   - Added batch support for the **ip** and **url** and **domain** commands.
+  - Fixed an issue Where DBotScore would create duplications in context. This will take effect from Demisto version 5.5.
```
Comment (amshamah419) [inline Integrations/integration-VirusTotal_CHANGELOG.md]:
```suggestion
  - Fixed an issue where the DBotScore would create duplications in the incident context. This effects Demisto version 5.5 and higher.
```

**Pull request merged**